### ISSUE TITLE
fix: 🤔 Screen Reader Does Not Announce Swipes on Next/Previous Buttons in sd-carousel on macOS Chromium/Firefox

### ISSUE THREAD

**Issue by Vahid1919** [solid-design-system/solid#1653]
## Current behavior
When using macOS with Chromium browsers, the screen reader does not announce swipes when the user presses the next and previous buttons in the sd-carousel. However, this issue does not occur on Safari or in Windows browsers with NVDA, where the swipes are announced correctly.

Also mentioned here: https://github.com/solid-design-system/solid/pull/1607#issuecomment-2447445856

## Expected behavior
VoiceOver screen reader announces slides and new pages when using the next and previous button. 

## Steps to reproduce

1. Navigate to https://solid-design-system.fe.union-investment.de/docs/?path=/docs/components-sd-carousel--docs on chromium/firefox based browsers
2. Activate VoiceOver and navigate sd-carousel on the Docs page.

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by coraliefeil** [comment]
Reach out to @mpantuso

**Comment by karlbaumhauer** [comment]
the ticket needs to stay in "to be refined" as the possible solution is not clear. Needs further investigation/communication with @matuzo and @Vahid1919 

@Vahid1919 any progress here?

**Comment by matuzo** [comment]
Yes, Vahid contacted me and I replied with my recommendation for testing [based on popular pairings](https://webaim.org/projects/screenreadersurvey10/#browsercombos) and my personal experience sorted by importance.

Test with: 

- JAWS with Chrome on Windows
- NVDA with Firefox on Windows
- VO with Safari on macOS/iOS
- Talkback with Chrome on Android

For your issue that means that you can definitely ignore VO with Firefox and you can expect that some things work differently or not properly when using VO with Chromium-based browser.

**Comment by yoezlem** [comment]
Solution:

Add a note to the specific component

**Comment by MartaPintoTeixeira** [comment]
In figma this is the branch to solve this issue: https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/branch/SHINcYB57gNcpsxrcsLenV/Component-Docs?node-id=2233-2413&p=f&t=URKuTwE2lUmedjoq-0

Design is ready. To be updated with real content from @smfonseca testing.

**Comment by smfonseca** [comment]
**NEXT/PREV**

TalkBack + Chrome => ❌
VO + Safari => ✅
VO + Firefox => ✅
VO + Chrome => ❌
Jaws + Chrome + Windows => no license to test
NVDA + Firefox + Windows => ❌

**AUTOPLAY**

TalkBack + Chrome => ✅
VO + Safari => ❌
VO + Firefox => only announces when "selected", not when "deselected"
VO + Chrome => only announces when "selected", not when "deselected"
Jaws + Chrome + Windows => no license to test
NVDA + Firefox + Windows => ✅

**Comment by smfonseca** [comment]
This is addressed together with the [overview pages](https://github.com/solid-design-system/solid/pull/2024) ticket. 
Table has been created in the carousel overview page.
